checks/collector.py

```diff
@@ -253,6 +253,8 @@ def run(self, checksd=None, start_event=True):
         """
         Collect data from each check and submit their data.
         """
+        log.info("Start of Collector.run()")
```

Comment:
I only left in logging to make it easier for reviewers to do testing, I will remove them in the final review.